packages/cli/README-source.md

```diff
@@ -1081,6 +1081,48 @@ This is an object that contains the data you returned from the `performSubscribe
 
 Read more in the [REST hook example](https://github.com/zapier/zapier-platform/blob/main/example-apps/rest-hooks/triggers/recipe.js).
 
+## BulkBundle Object
+
+*Added in v15.8.0.*
+
+This object holds a user's auth details and the bulked data for the API requests. It is used only with a `create` action's `performBulk` function.
+
+> The `bulkbundle` object is passed into the `performBulk` function as the second argument - IE: `performBulk: (z, bulkbundle) => {}`.
+
+### `bulkbundle.authData`
+
+It is a user-provided authentication data, like `api_key` or `access_token`. [Read more on authentication.](#authentication)
+
+### `bulkbundle.groupedBy`
+
+It is a user-provided data for a set of selected [`inputFields`](#input-fields) to group the multiple runs of a `create` action by.
+
+### `bulkbundle.bulk`
+
+It is a list of objects of user-provided data and some meta data to allow multiple runs of a `create` action be processed in a single API request.
+
+#### `bulkbundle.bulk[].inputData`
+
+It is a user-provided data for a particular run of a `create` action in the bulk, as defined by the [`inputFields`](#input-fields).
+
+#### `bulkbundle.bulk[].meta`
+
+It contains an idempotency `id` provided to the `create` action to identify each run's data in the bulk data.
+
+## Bulk-Write in Create Actions
```

Comment:
I think this section is better placed before `## Output Fields` and after `### Nested & Children (Line Item) Fields`. It can still be a level-two heading, but I suggest we say `## Bulk Create Actions`.

Reply:
That would make it be in between section `## Input Fields` and `## Output Fields`. Please, why so?

Reply:
Sorry I had a typo there. It should be before `## Z Object` and after `### Nested & Children (Line Item)`. Something like:

<img src="https://cdn.zappy.app/d695af7bf9a6f259320394e59b20b5aa.png" width="400">

Reply:
Done, thanks